containers/sandbox/Dockerfile

```diff
@@ -30,8 +30,14 @@ RUN mkdir -p -m0755 /var/run/sshd
 # symlink python3 to python
 RUN ln -s /usr/bin/python3 /usr/bin/python
 
-# install basic dependencies for CodeActAgent
-RUN pip3 install --upgrade pip
-RUN pip3 install jupyterlab notebook jupyter_kernel_gateway flake8
-# TODO: those dependencies are needed for agentskills, we should pack them in a new sandbox image
-RUN pip3 install python-docx PyPDF2 python-pptx pylatexenc openai opencv-python
+# ==== OpenDevin Runtime Client ====
+RUN mkdir -p /opendevin && mkdir -p /opendevin/logs && chmod 777 /opendevin/logs
+RUN wget "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-$(uname)-$(uname -m).sh"
+RUN bash Miniforge3-$(uname)-$(uname -m).sh -b -p /opendevin/miniforge3
+RUN bash -c ". /opendevin/miniforge3/etc/profile.d/conda.sh && conda config --set changeps1 False && conda config --append channels conda-forge"
+RUN echo "export PATH=/opendevin/miniforge3/bin:$PATH" >> ~/.bashrc
```

Comment:
Does this means miniforge3's python will be the default?

Reply:
I directly copy the dockerfile in #2101. I guess yes, minforge3 python will be default.